### DISCUSSION TITLE
Embedding Power BI reports

### DISCUSSION THREAD

**Discussion by namas191297** [Budibase/budibase#7320] [Help] [answered]
Hello,

I was wondering if Power BI reports could be embedded into Budibase somehow. I have tried using Embed but either it doesn't have the functionality yet or I am doing something wrong. Any help would be appreciated :).

**Comment by melohagan** [comment, marked as answer]
Hi @namas191297 

Currently the embed does not support importing libraries.

Good news is that the dev team is currently working on custom components. 
This will support NodeJS libraries, thus a Power BI component could be built: https://powerbi.microsoft.com/en-us/blog/pbie-nodejs-sdk/

That release is scheduled for the end of September if it all goes smoothly!
